api/BaseAudioContext.json

```diff
@@ -113,15 +99,20 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/BaseAudioContext/createAnalyser",
           "support": {
-            "chrome": {
-              "version_added": "10",
-              "prefix": "webkit"
-            },
+            "chrome": [
+              {
+                "version_added": "56"
+              },
+              {
+                "version_added": "14",
+                "prefix": "webkit"
```

Comment:
I'm pretty sure there's never been a `webkitCreateAnalyser` method anywhere as this would imply. The prefixed things have been `webkitAudioContext`, `webkitAudioPannerNode` and `webkitOfflineAudioContext`.

I presume this change wasn't suggested by mdn-bcd-collector?